pkg/agent/client/client_test.go

```diff
@@ -696,6 +824,19 @@ func TestFetchJWTSVID(t *testing.T) {
 			},
 			err: "JWTSVID issued after it has expired",
 		},
+		{
+			name: "grpc call to NewJWTSVID fails",
+			setupTest: func(err error) {
+				tc.svidClient.jwtSVID = &types.JWTSVID{
+					Token:     "token",
+					ExpiresAt: issuedAt,
+					IssuedAt:  expiresAt,
```

Comment:
These fields look mixed up, is it intentional?

Reply:
Thanks. I forgot to change it after copying the previous test. Fixed at 7488427c